sdks/python/apache_beam/examples/snippets/snippets_test.py

```diff
@@ -245,10 +245,12 @@ def even_odd(x):
 class TypeHintsTest(unittest.TestCase):
 
   def test_bad_types(self):
-    p = TestPipeline()
+    # [START type_hints_missing_define_numbers]
+    p = TestPipeline(
+        options=TypeOptions.from_dictionary({'pipeline_type_check': True})
```

Comment:
This could be `PipelineOptions(pipeline_type_check=True)`, right? (Same elsewhere.)